src/openrct2/CommandLineSprite.cpp

```diff
@@ -446,7 +446,15 @@ namespace OpenRCT2
                 }
 
                 path = fs::u8path(path).generic_u8string();
-                fprintf(stdout, "{ \"path\": \"%s\", \"x\": %d, \"y\": %d },\n", path.c_str(), g1.x_offset, g1.y_offset);
+                if (spriteIndex + 1 != maxIndex)
+                {
+                    fprintf(stdout, "{ \"path\": \"%s\", \"x\": %d, \"y\": %d },\n", path.c_str(), g1.x_offset, g1.y_offset);
+                }
+                else
+                {
+                    // Do not print the trailing comma
+                    fprintf(stdout, "{ \"path\": \"%s\", \"x\": %d, \"y\": %d }\n", path.c_str(), g1.x_offset, g1.y_offset);
+                }
```

Comment:
This really seems like unnecessary duplication. Branch on ",\n" vs "\n" only, not the entire formatting string.

However, I think we should consider refactoring this to use the JSON library we already include.

Reply:
Yeah I realize that, but honestly this seemed like the easier-to-read alternative

> However, I think we should consider refactoring this to use the JSON library we already include.

Oh! That seems like the _best_ alternative altogether, then. If this ends up being the approach settled on, this PR can be closed.

Reply:
> Yeah I realize that, but honestly this seemed like the easier-to-read alternative

I don't follow. It could be trimmed down to something as short as this:

```suggestion
                fprintf(stdout, "{ \"path\": \"%s\", \"x\": %d, \"y\": %d }", path.c_str(), g1.x_offset, g1.y_offset);
                fprintf(stdout, (spriteIndex + 1 != maxIndex) ? ",\n" : "\n");
```

Reply:
You're right... I didn't even consider using a ternary in this case

Reply:
@AaronVanGeffen Do you still want to move this over to our JSON library, or is this good enough?

Reply:
I'm also curious about this.

Reply:
@Gymnasiast I think that's not necessary for this PR. Not sure if we print JSON manually in more places, but ideally we refactor all of it in one swoop in a future PR.